test/extended/networking.sh

```diff
@@ -256,13 +300,12 @@ else
   # Docker-in-docker is not compatible with selinux
   disable-selinux
 
-  os::log::info "Ensuring that previous test cluster is shut down"
-  ${CLUSTER_CMD} stop
-
-  test-osdn-plugin "subnet" "redhat/openshift-ovs-subnet" "false"
+  # Ignore deployment errors for a given plugin to allow other plugins
+  # to be tested.
+  test-osdn-plugin "subnet" "redhat/openshift-ovs-subnet" "false" || true
```

Comment:
Whenever a `|| true` shows up, the question must be asked -- why is failure here something we should tolerate without any sort of action acknowledging it?